app/Http/Requests/Api/Auth/RegisterRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Requests\Api\Auth;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RegisterRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'email' => 'required|string|email|max:255|unique:users',
+            'password' => 'required|string|min:6|confirmed|regex:/^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{6,}$/',
+            'phone_number' => 'required|string',
+            'dob' => 'required|date',
+            'image' => 'sometimes|image|mimes:png,jpg,jpeg',
+            'id_img' => 'string',
+            'gender' => 'required|string|in:male,female'
+        ];
```

Comment:
address attribute is missing here in the validation and I think it's missing in the database too. So please add it to the DB and then consider using it in the rest of operations.

Reply:
Resolved,

Just I am recieving `SQLSTATE[23000]: Integrity constraint violation: 19 FOREIGN KEY constraint failed (SQL: DROP TABLE users)` when refreshing the testing database **sqlite**